Assets/UnityDependency/Tests/Editor/Defaults/DefaultPathAbsolute.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace UnityDependency.Tests
+{
+    using Alquimiaware;
+    using NUnit.Framework;
+    using UnityEngine;
+
+    public abstract partial class DependencyExtension_ParentChain
+    {
+        public class DefaultPathAbsolute : DependencyExtension_ParentChain
+        {
+            [Test]
+            [ExpectedException(typeof(System.ArgumentOutOfRangeException))]
```

Comment:
Shoudl I check the exception itself somehow? Verify it is thrown and that the contents seem to be what is expected.